src/libraries/Microsoft.PowerFx.Connectors/OpenApiParser.cs

```diff
@@ -113,7 +113,7 @@ internal static IEnumerable<ConnectorFunction> GetFunctionsInternal(ConnectorSet
         {
             bool connectorIsSupported = true;
             string connectorNotSupportedReason = string.Empty;
-            List<ConnectorFunction> functions = new ();
+            List<ConnectorFunction> functions = new();
```

Comment:
>List<ConnectorFunction> functions = new(); [](http://example.com/codeflow?start=12&length=42)

If there is no actual change, better to avoid this spacing change.